schemas/com.snowplowanalytics.telemetry/collector_telemetry/jsonschema/1-0-0

```diff
@@ -0,0 +1,60 @@
+{
+  "$schema": "http://iglucentral.com/schemas/com.snowplowanalytics.self-desc/schema/jsonschema/1-0-0#",
+  "description": "Schema for collector telemetry",
+  "self": {
+    "vendor": "com.snowplowanalytics.telemetry",
+    "name": "collector_telemetry",
+    "format": "jsonschema",
+    "version": "1-0-0"
+  },
+  "type": "object",
+  "properties": {
+    "collectorHost": {
+      "type": "string",
+      "description": "Host URL of the collector, e.g. https://collector.acme.com",
+      "format": "uri"
+    },
+    "collectorPath": {
+      "type": "string",
+      "description": "Path of the collector, e.g. /com.snowplowanalytics.snowplow/tp2"
+    },
+    "appId": {
+      "type": [
+        "string",
+        "null"
+      ],
+      "description": "The Application Identifier of the tracker",
+      "maxLength": 255
+    },
+    "pageUrl": {
```

Comment:
Let's remove this - there's too much potential to capture apps that include sensitive tokens / data as part of URL parameters.

Reply:
In the Chrome extension at least, we only capture the page hostname which should avoid most of those issues. Perhaps we can have that here as well, rather than removing it entirely?

Reply:
I've added a `pageHost` property to this schema.